app/models/manageiq/providers/vmware/cloud_manager/refresh_parser.rb

```diff
@@ -56,6 +57,10 @@ def get_vms
     process_collection(@inv[:vms], :vms) { |vm| parse_vm(vm) }
   end
 
+  def get_stacks
+    process_collection(@inv[:vapps], :orchestration_stacks) { |vapp| parse_stack(vapp) }
+  end
```

Comment:
We already have get_vapps so we don't need a separate get_stacks method, just drop your `process_collection` after the code that retrieves the vapps.